src/lib/rule-finder.js

```diff
@@ -72,13 +72,17 @@ function _getPluginRules(config) {
 function _getAllAvailableRules(pluginRules) {
   var allRules = fs
     .readdirSync('./node_modules/eslint/lib/rules')
-    .map(function removeJsFromFilename(filename) {
-      return filename.replace(/\.js$/, '')
-    })
+  var filteredAllRules = []
+  allRules.forEach(function filterAllRules(filename) {
+    /* istanbul ignore next */
```

Comment:
If coverage is running on not-the-latest eslint such that this ignore is needed, could the tests do something like `touch ./node_modules/eslint/lib/rules/test.yml` to ensure it's covered?